variables.tf

```diff
@@ -3,53 +3,53 @@
 ##############################################################################
```

Comment:
These variables.tf files are where you want to make all your suggestions. They will get regenerated in the readme files wherever the `inputs` section exists.